sdk/tests/conformance2/reading/read-pixels-into-pixel-pack-buffer.html

```diff
@@ -202,19 +219,34 @@
 function testPackAlignment()
 {
     debug("");
-    debug("Verify that reading pixels to PIXEL_PACK buffer works fine with various PACK_ALIGNMENT values.");
-
-    runTestIteration(1, 1, 2);
-    runTestIteration(2, 1, 2);
-    runTestIteration(4, 1, 2);
-    runTestIteration(8, 1, 2);
-    runTestIteration(4, 2, 2);
-    runTestIteration(8, 2, 2);
-    runTestIteration(4, 3, 2);
-    runTestIteration(8, 3, 2);
-    runTestIteration(1, 0, 0);
-    runTestIteration(4, 1, 1);
-    runTestIteration(8, 1, 1);
+    debug("Verify that reading pixels to PIXEL_PACK buffer works fine with various pack alignments.");
+    runTestIteration(1, 3, 1, 0, 0, 0);
```

Comment:
Good suggestion.  I will revise in a followup CL, so it's clearer what's been done.